lib/index.ts

```diff
@@ -60,6 +60,7 @@ export { getRegistryAndName } from './utils';
 export { BalenaYml, ParsedBalenaYml };
 export { PathUtils };
 export { ResolveListeners };
+export { CANONICAL_HUB_URL } from './constants';
```

Comment:
What's the need to make this constant public API?

Reply:
This ended up as part of some refactoring that sort of took place as I was going.  It is used in the builder.